### PR TITLE
Sort keybindings by translated name

### LINKED ISSUE #179: Sorted keybindings
Btw, while your'e at at...the list of keybindings becomes pretty big. Can we maybe [sort them alphabetically by name](https://docs.godotengine.org/en/stable/classes/class_array.html#class-array-method-sort-custom)? This way all the "Pan" actions would be grouped together for example and stuff is easier to find in general.

Edit: Ah i just see you use an unordered map/dictionary for this. The order is different everytime Lorien is started. Probably a good idea to convert to an array sort it and then build the UI list from that. 

Edit: Nevermind it's already an Array built from the InputMap. So it would only need to be sorted.

_Originally posted by @mbrlabs in https://github.com/mbrlabs/Lorien/issues/177#issuecomment-1179508344_

### PR DESCRIPTION
**Pull request by MrApplejuice**
Sorts the keybindings by their translated string.

Fixes #179

### CODE REVIEW

File: lorien/UI/Dialogs/KeyBindingsList.gd [hunk 1]
```diff
@@ -13,20 +13,29 @@ func _ready() -> void:
 	_add_key_dialog.connect("hide", self, "_bind_key_dialog_hidden")
 	GlobalSignals.connect("language_changed", self, "_populate_input_list")
 
+# -------------------------------------------------------------------------------------------------
+func _second_element(a: Array, b: Array) -> bool:
```
Comment (mbrlabs) [inline lorien/UI/Dialogs/KeyBindingsList.gd:17]: Could you change that to `_compare_second_element`?

Comment (MrApplejuice) [reply lorien/UI/Dialogs/KeyBindingsList.gd:17]: Sorry, a bit slow to respond right now. Lot's of stuff on my place ;-)

Comment (mbrlabs) [reply lorien/UI/Dialogs/KeyBindingsList.gd]: Don't worry, no problem. Thanks!

**Comment by mbrlabs** [comment]
Thanks!

**Pull request merged**